query/encode.go

```diff
@@ -217,6 +217,16 @@ func reflectValue(values url.Values, val reflect.Value, scope string) error {
 			continue
 		}
 
+		if sv.Kind() == reflect.Map {
+			// continue
+			// fmt.Println(name, sv.Len(), sv.Interface().(map[string]string)["aaa"])
+			for k, v := range sv.Interface().(map[string]string) {
+				values.Add(fmt.Sprintf("%s[%s]", name, k), v)
+			}
+			// values.Add()
```

Comment:
It looks like the commented out code here is a relic from development/debugging. It should be removed in the final PR, if this is going to be merged into `master` branch.